instrumentation/opentelemetry-annotations-1.0/javaagent/build.gradle.kts

```diff
@@ -2,6 +2,14 @@ plugins {
   id("otel.javaagent-instrumentation")
 }
 
+muzzle {
```

Comment:
Should we add muzzle after publishing? Just in case it fails, it doesn't happen because of a release but because of a PR

Reply:
Hmm, good point. I'll remove that for now and add a TODO to add it back after the release.